src/components/Country/Country.module.css

```diff
@@ -0,0 +1,24 @@
+.countryContainer {
+  display: flex;
+  height: 50px;
+  align-items: center;
+  gap: 10px;
+  padding: 0 10px;
+}
+
+.countryContainer:nth-child(odd) { background-color: #5582e1; }
+.countryContainer:nth-child(even) { background-color: #4169af; }
+
+.img {
+  width: 40px;
+  height: 50%;
+  border: 1px solid white;
+}
```

Comment:
- [ ] The Images of every country can be improved as it seems a little distorted.
![Screen Shot 2022-02-02 at 19 26 46](https://user-images.githubusercontent.com/67211919/152266508-9d891936-8857-4bb5-a525-1e57d505d352.png)

Reply:
Done